examples/resnet/resnet_main.py

```diff
@@ -122,6 +114,7 @@ def get_weights(self):
 @ray.remote
 class ResNetTestActor(object):
     def __init__(self, data, dataset, eval_batch_count, eval_dir):
+        os.environ["CUDA_VISIBLE_DEVICES"] = ""
```

Comment:
Is this necessary? We do want to run on gpus.

Reply:
The test actor is currently running in the background during training, so having a dedicated gpu to it isn't strictly necessary. We could have a flag to set whether the test actor uses a gpu, but I think it is unnecessary.